.jenkins/Jenkinsfile

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/groovy
+@Library('jenkins-pipeline@v0.4.5')
+import com.invoca.docker.*;
+pipeline {
+  agent {
+    kubernetes {
+      defaultContainer "ruby"
+      yamlFile ".jenkins/ruby_build_pod.yml"
+    }
+  }
+
+  environment {
+    GITHUB_TOKEN = credentials('github_token')
+    GITHUB_KEY = credentials('github_key')
+    BUNDLE_GEM__FURY__IO = credentials('gemfury_deploy_token')
+  }
+
+  stages {
+    stage('Setup') {
+      steps {
+        updateGitHubStatus('clean-build', 'pending', 'Unit tests.')
+        script {
+          sh 'bundle install'
+        }
+      }
+    }
+    stage('Unit Test') {
+      steps {
+        script {
+          sh '''
+            # get SSH setup inside the container
+            eval `ssh-agent -s`
+            echo "$GITHUB_KEY" | ssh-add -
+            mkdir -p /root/.ssh
+            ssh-keyscan -t rsa github.com > /root/.ssh/known_hosts
+            bundle install
```

Comment:
this needs to run the tests, you have bundle install here a second time :) 

Likely `bundle exec rake` but I didn't check